projects/hslayers/README.md

```diff
@@ -13,10 +13,15 @@ HSLayers-NG is a library which extends OpenLayers 6 functionality by providing a
 
 ## Demo
 
-Check out the examples to get an idea:  
+Check out the examples and live projects to get an idea:  
 * http://ng.hslayers.org/examples/
-* http://opentransportnet.eu/create-maps
+* https://groundwater.smartagro.lv/fie/index.html
+* https://www.smartafrihub.com/cs/map1/
 * http://sdi4apps.eu/spoi
+* (Material) https://www.smartafrihub.com/cs/best-practises
```

Comment:
I would replace this link with https://atlasbestpractices.com, which opens the map directly.

Reply:
Ok